blog/content/post/2020-04-26-measuring_memory_usage_rte.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Measuring memory and CPU usage on Yocto & Armbian
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/rte-new-revision.jpg
+author: piotr.konkol
+layout: post
+published: true
+date: 2020-04-26
+archives: "2020"
+
+tags:
+  - measurment
+  - rte
+  - yocto
+  - armbian
+  - ram
+categories:
+  - IoT
+  - Miscellaneous
+
+---
+
+## Intro
+This article will show you how to perform and interpret measurements of some of
+embedded systems computational resources. Case covered here will be an attemt
+to resolve a question whether our configuration will be enough for the given
+tasks, or an upgrade of the hardware specification will be necessary. As IoT and
+embedded systems often get very big in terms of device number, it is necessary
+to perfrom such analysis and match the specification exactly with the planned
+workload and the profit from such action will show as the system grows larger.
+
+Hardware configuration we will use is based on [RTE](https://3mdeb.com/products/open-source-hardware/rte/)
```

Comment:
@pkonkol based? Or it is exactly RTE?

Reply:
Well, there is also the DUT, which is not relevant but required for regression.

Reply:
@pietrushnic I thought about this in a way that @lwcislo mentioned